Resources/Prototypes/_EstacaoPirata/Mobs/Species/Daengati.yml

```
@@ -35,16 +41,24 @@
     heatDamage:
       types:
         Heat : 0.5 #O padrão é 1.5
+#  - type: Daengati # Esta dando erro então vou deixar removido.. tem que adicionar depois
+#    modifiers:
+#      coefficients:
+#        Heat: 0.6
+#        Cold: 1.3
+#        Piercing: 1.15
   - type: Damageable
     damageModifierSet: Daengati
-  - type: NightVision # Gaby edit
-    isActive: false
-    toggleAction: ToggleDaengatiNightVision
-    color: "#9c452a"
-    strength: 2
-    activateSound: null
-    deactivateSound: null
-
+  - type: Body
+    prototype: Daengati
+  - type: TypingIndicator
+    proto: oni #Mudar apos adicionar indicador de escrita de daengati
+#  - type: tags # Esta dando erro então vou deixar removido.. tem que adicionar depois
+#    tags:
+#    - CanPilot
+#    - FootstepSound
+#    - DoorBumpOpener
+#    - AnomalyHost
```

Isso aqui não pode tirar. Vou reproduzir os erros.

---

```suggestion
  - type: Tag
    tags:
    - CanPilot
    - FootstepSound
    - DoorBumpOpener
    - AnomalyHost
```

só um typo no nome do component.